External/FEXCore/Source/Interface/IR/Passes/ConstProp.cpp

```diff
@@ -194,7 +194,8 @@ bool ConstProp::Run(OpDispatchBuilder *Disp) {
       case OP_ZEXT: {
         auto Op = IROp->C<IR::IROp_Zext>();
         uint64_t Constant;
-        if (Disp->IsValueConstant(Op->Header.Args[0], &Constant)) {
+        if (Op->SrcSize != 64 &&
+            Disp->IsValueConstant(Op->Header.Args[0], &Constant)) {
```

Comment:
We don't currently have a clean way to push a constant in to the FPR space to handle that situation, so it would end up looking the same right now.
I think we should look forward to adding an IR op for AArch64's FMOV(immediate) function to optimize that path.